locales-src/asrouter.ftl

```diff
@@ -90,17 +90,42 @@ cfr-whatsnew-button =
   .label = What’s New
   .tooltiptext = What’s New
 
+cfr-whatsnew-fx70-hero-title = { -brand-short-name } now fights harder for your privacy
+cfr-whatsnew-fx70-hero-body = The latest update enhances the Tracking Protection feature and makes it easier than ever to create secure passwords for every site.
```

Comment:
Side note: Fluent supports multiline strings, in case you don't want to have super long lines of text.

```
 cfr-whatsnew-fx70-hero-body = 
    The latest update enhances the Tracking Protection feature and makes it 
    easier than ever to create secure passwords for every site.
```

Reply:
Indented to 3 spaces and wrapped 80 characters